addons/mail/static/tests/tours/mail_activity_schedule_from_chatter.js

```diff
@@ -8,6 +8,25 @@ registry.category("web_tour.tours").add("mail_activity_schedule_from_chatter", {
         {
             trigger: "button:contains('Activities')",
         },
+        {
+            trigger: "input[id='activity_type_id_0']",
+            run: "click",
+        },
+        {
+            trigger: ".dropdown-item:contains('Call')",
+        },
+        {
+            extra_trigger: "button:contains('Schedule')",
+            trigger: "input[id='activity_type_id_0']",
+            run: function (action_helper) {
+                setTimeout(() => {
+                    action_helper.click();
+                }, 100);
+            },
```

Comment:
Nitpicking: could be shorter:
```js
run(action_helper) {
    setTimeout(() => action_helper.click(), 100);
},
```

The added value is that there's less LOCs, so less scrolling needed to read the tour steps :)